### PR TITLE
Fix incorrectly formatted version rules

### PR DESCRIPTION
**Pull request by jphan32**
## Related issue number
- Issue: #131

## Description
Fix incorrectly formatted version rules.
- Remove the .* at line18 numpy>=1.19.*

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
     install_requires=[
         "boltons>=20.2.1",
         "numpy==1.19.*;python_version<'3.7'",
-        "numpy>=1.19.*;python_version>='3.7'",
+        "numpy>=1.19;python_version>='3.7'",
         "scipy==1.5.*;python_version<'3.7'",
```
Comment (RonnyPfannschmidt) [inline setup.py:46]: i believe the scipy dependency is in need of a change as ell

**Comment by jphan32** [comment]
Could someone please review my PR? Thank you.

**Comment by jphan32** [comment]
Hello maintainers,

I recently submitted a pull request and noticed that there was a build error during the review process. Upon inspecting the error message, it was related to the install_requires section in the setup.py file. Specifically, the error message was:

```
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      error in torchsde setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; .* suffix can only be used with `==` or `!=` operators
          numpy>=1.19.*;python_version>='3.7'
               ~~~~~~~^
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error
```

To address this, I updated the requirement string from **numpy>=1.19.*;**python_version>='3.7' to **numpy>=1.19;**python_version>='3.7' and committed the changes. However, the build test still seems to produce the same error message indicating the presence of the older code.

Could you please help me understand why this might be happening? Is there something I may have missed or another step I need to take to ensure the updates are reflected?

Thank you for your assistance!

**Comment by patrick-kidger** [comment]
Ah, that's because it's pulling from the GitHub main branch to do a version check. Let me poke at that.

FWIW you may note that torchsde hasn't been touched in a couple of years; I'll try to get this PR merged, but I'd recommend [Diffrax](https://github.com/patrick-kidger/diffrax) for new projects.

**Comment by lxuechen** [comment]
@patrick-kidger I enabled force push for you. Can you check if you can force merge?

**Comment by lxuechen** [comment]
> @patrick-kidger I enabled force push for you. Can you check if you can force merge?

Merged and thanks, @jphan32. Let me know what else I should help with.

**Comment by jphan32** [comment]
I've confirmed that my PR has been successfully merged into the master branch. Thank you. Also, I appreciate the introduction to Diffrax.

**Pull request merged**